src/components/NavTablet/NavTablet.module.css

```diff
@@ -0,0 +1,33 @@
+.navTablet {
+  display: none;
+}
+
+@media screen and (min-width: 765px) {
+  .navTablet {
+    height: 81px;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 16px 69px 16px 32px;
+    position: fixed;
+    background-color: var(--primary-background-color);
+    z-index: 999;
+  }
+
+  .navTablet button {
+    font-size: 1rem;
+  }
```

Comment:
tutaj bym dodał width: 35px i display: flex z align-items:center to wtedy button będzie jak na figmie :D

Reply:
Fixed. O teraz super wygląda i nie trzena już poprawiać :D Dzięki!